### ISSUE TITLE
Unet dropout

### ISSUE THREAD

**Issue by luistelmocosta** [qubvel/segmentation_models.pytorch#205]
Hello, I would like to add test time dropout to my segmentation pipeline. I am using Unet segmentation model and I noticed that there isn't any dropout in the SegmentationHead. Is there any way to use dropout with that module? Any workaround so it would be possible to apply dropout in test time?

Kind regards

**Comment by howard-mahe** [comment]
Hello, there is already a dropout (default: 0.2) in the segmentation head of the UNet architecture (see [base/heads.py#L16](https://github.com/qubvel/segmentation_models.pytorch/blob/a3cc9ac7b3430967414506f37c5f043e1ba3397f/segmentation_models_pytorch/base/heads.py#L16)).

```python
import torch.nn as nn

def enable_dropout(module):
  if isinstance(module, isinstance(a, nn.modules.dropout._DropoutNd)):
    module.train()

# put the model in eval mode with test-time dropout
model.eval()
model.apply(enable_dropout)
```

**Comment by luistelmocosta** [comment]
The linked you post has only dropout for the `ClassificationHead`.

**Comment by howard-mahe** [comment]
You are right, my bad!

In the Bayesian Classifier of the Bayesian SegNet paper, the authors "insert dropout after the last decoder unit, before the classifier" which corresponds to:
```python
class SegmentationHead(nn.Sequential):

    def __init__(self, in_channels, out_channels, kernel_size=3, dropout=None, activation=None, upsampling=1):
        dropout = nn.Dropout(p=dropout, inplace=True) if dropout else nn.Identity()
        conv2d = nn.Conv2d(in_channels, out_channels, kernel_size=kernel_size, padding=kernel_size // 2)
        upsampling = nn.UpsamplingBilinear2d(scale_factor=upsampling) if upsampling > 1 else nn.Identity()
        activation = Activation(activation)
        super().__init__(dropout, conv2d, upsampling, activation)
```
and then modify the call to it in [unet/model.py#L75](https://github.com/qubvel/segmentation_models.pytorch/blob/a3cc9ac7b3430967414506f37c5f043e1ba3397f/segmentation_models_pytorch/unet/model.py#L75) in order to apply dropout.
